internal/docs/man.go

```diff
@@ -198,6 +198,16 @@ func manPrintJSONFields(buf *bytes.Buffer, command *cobra.Command) {
 	buf.WriteString("\n")
 }
 
+func manPrintExitCodes(buf *bytes.Buffer) {
+	buf.WriteString("# EXIT CODES\n")
+	buf.WriteString("0\n:   Success\n\n")
+	buf.WriteString("1\n:   Error\n\n")
+	buf.WriteString("2\n:   Command canceled\n\n")
+	buf.WriteString("4\n:   Authentication required\n\n")
+	buf.WriteString("8\n:   Checks pending\n\n")
```

Comment:
I don't think this belongs here, I think it belongs with the `gh pr checks` command only, as that's the only command that returns it. This falls into the following "NOTE: Specific commands may have additional exit codes. Refer to the command's help for more information" string